prql-compiler/src/sql/std.sql.prql

```diff
@@ -27,8 +27,8 @@ let count_distinct = column -> s"COUNT(DISTINCT {column:0})"
 # Window functions
 let lag = offset column -> s"LAG({column:0}, {offset:0})"
 let lead = offset column -> s"LEAD({column:0}, {offset:0})"
-let first = offset column -> s"FIRST_VALUE({column:0}, {offset:0})"
-let last = offset column -> s"LAST_VALUE({column:0}, {offset:0})"
+let first = column -> s"FIRST_VALUE({column:0})"
+let last = column -> s"LAST_VALUE({column:0})"
```

Comment:
Maybe the `RESPECT NULLS` and `IGNORE NULLS` options are needed?

> The SQL standard defines a RESPECT NULLS or IGNORE NULLS option for lead, lag, first_value, last_value, and nth_value. This is not implemented in PostgreSQL: the behavior is always the same as the standard's default, namely RESPECT NULLS. Likewise, the standard's FROM FIRST or FROM LAST option for nth_value is not implemented: only the default FROM FIRST behavior is supported. (You can achieve the result of FROM LAST by reversing the ORDER BY ordering.)

https://www.postgresql.org/docs/current/functions-window.html

Reply:
WDYM by "needed"? as in they'd be helpful or we need to add them now?

Reply:
Sorry, of course there is no need to add it now, a follow up issue.
But I think this option is very important and related to #2622.

Reply:
OK great! Sorry for misunderstanding at first